code-challenges/challenge-04/linked-list.js

```diff
@@ -0,0 +1,48 @@
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next || null;
+  }
+}
+
+class LinkedList {
+  constructor(value) {
+    this.head = null;
+    this.length = 0;
+    this.insert(value);
+  }
+
+  insert(value) {
+    //adds new node with that value at the head
+    const newNode = new Node(value, this.head);
+    newNode.next = this.head;
+    this.head = newNode;
+    this.length++;
+    return this;
+  }
+
+  includes(value) {
+    let thisNode = this.head;
+
+    while(thisNode) {
+      if(thisNode.value === value) {
+        return true;
+      } else return false;
```

Comment:
returning false too soon. Right now if the first item (this.head) isn't what you are looking for you return false